app/stacks/front-door/locals.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  frontend_mappings = {
+    national_infrastructure_frontend = {
+      name              = "pins-fd-${local.service_name}-${local.resource_suffix}"
+      frontend_endpoint = "pins-fd-${local.service_name}-${local.resource_suffix}.azurefd.net"
+      patterns_to_match = ["/*"]
+    }
+  }
+  service_name    = "front-door"

```

Comment:
I think this should be "common". It's not part of the common stack, but it's still a common resource I'd say.

Reply:
This will require public DNS changes as it'll affect the azurefd.net address of the Front Door. Can we address this later after testing?

Reply:
Yeah sounds good to me